From 2a30b8254d34a2656d798931f46af157afd152c0 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 18 Aug 2021 17:26:49 +0100
Subject: [PATCH] overlays: rpi-poe-plus: Improve the cooling levels

The PoE HAT cooling levels are not well suited for the PoE+ HAT - the
fan fails to come on until the temperature reaches the third trip point
(50C).

Give the rpi-poe-plus overlay a different set of cooling levels -
0 32 64 128 255, as suggested by @chris-kai-in.

See: https://github.com/raspberrypi/firmware/issues/1607

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/rpi-poe-plus-overlay.dts | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm/boot/dts/overlays/rpi-poe-plus-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-poe-plus-overlay.dts
@@ -17,3 +17,7 @@
 		};
 	};
 };
+
+&fan0 {
+	cooling-levels = <0 32 64 128 255>;
+};
